google_guest_agent/addresses_unix.go

```diff
@@ -29,3 +30,11 @@ func addIPForwardEntry(route ipForwardEntry) error {
 func getIPForwardEntries() ([]ipForwardEntry, error) {
 	return nil, errors.New("getIPForwardEntries unimplemented on non Windows systems")
 }
+
+func addAddress(ip net.IP, mask net.IPMask, index uint32) error {
+	return nil
```

Comment:
is this a stub? why do other stubs return errors but this returns nil?

Reply:
Copy paste of an existing function and I didn't add the error like i did for others. I can add it now if you want

Reply:
yeah let's be consistent there, at least for readability